djangocms_pageadmin/cms_toolbars.py

```diff
@@ -20,7 +20,8 @@ def change_admin_menu(self):
         super().change_admin_menu()
         menu = self.toolbar.get_menu(ADMIN_MENU_IDENTIFIER)
         item = menu.items[0]
-        if item and item.url and "admin/cms/pagecontent/" in item.url:
+        if item and getattr(item, 'url', None) and "admin/cms/pagecontent/" in item.url:
+            # Handle menu items like SubMenu that have no url attribute.
```

Comment:
May sound picky but the comment look to be addressing the line below it not the if clause above, can you move the comment to above the if clause please?

Reply:
This is the kind of pickyness I can get behind :D